questionnaires/technology_assessment_measurement_fast_form/technology_assessment_measurement_fast_form_armt.json

```diff
@@ -0,0 +1,866 @@
+[
+    {
+        "field_name": "tam_usefulness_1",
+        "form_name": "technology_assessment_measurement_fast_form",
+        "section_header": "Please think about all aspects of the RADAR system (the apps and wearable fitness tracker) and answer the following questions.<br><br>Usefulness",
```

Comment:
Maybe this can be modified as well? The app doesn't support HTML in `section_header`.